src/App.css

```diff
@@ -32,6 +32,17 @@ body,
 	max-width: 400px;
 }
 
+.descope-saml-container {
```

Comment:
I would not name it `saml-container`, we can do something more generic that will fit other wide flows, maybe `descope-wide-container`